pages/hosted_private_cloud/hosted_private_cloud_powered_by_vmware/vmware_iam_getting_started/guide.fr-fr.md

```diff
@@ -0,0 +1,138 @@
+---
+title: "Guides premiers pas avec IAM dans mon cloud privée VMware managé par OVHcloud"
+excerpt: "Comment fonctionne IAM dans un cloud privé VMware managé par OVHcloud et une faq"
+updated: 2024-05-24
+---
+
+<style>
+.w-400 {
+  max-width:400px !important;
+}
+.h-600 {
+  max-height:600px !important;
+}
+</style>
+
+
+# Table des matières
+1. [Objectif](#Objectif)
+2. [Etape 1 - En pratique](#Enpratique)
+3. [Etape 2 - FAQ](#FAQ)
+4. [A suivre - Comment activer IAM](#Asuivre)
+5. [Aller plus loin](#Allerplusloin)
+
+---
+## Objectif
+**Dans ce guide, nous allons vous expliquer comment démarrer avec IAM dans votre Cloud Privée VMware sur OVHcloud et voir quelques questions réponses dans la FAQ** 
+
+---
+## Etape 1 : En pratique
+
+## Premiers pas avec IAM et mon cloud privée VMware managé par OVHcloud
+
+Pour utiliser IAM dans votre cloud privé, il va faut : 
+
+- [x] Activer IAM.
```

Comment:
Not a valid format

Reply:
Not valid at all, this is pure Markdown ?